quesma/testdata/clients/clover.go

```diff
@@ -0,0 +1,350 @@
+					},
+					"meta": {
+						"indexPatternString": "ab*",
+						"intervalString": "900000ms",
+						"normalized": true,
+						"panelId": "0",
+						"seriesId": "1",
+						"timeField": "@timestamp"
+					}
+				}
+			},
+			"query": {
+				"bool": {
+					"filter": [],
+					"must": [
+						{
+							"range": {
+								"@timestamp": {
+									"format": "strict_date_optional_time",
+									"gte": "2024-10-11T09:58:03.723Z",
+									"lte": "2024-10-11T10:13:03.723Z"
+								}
+							}
+						},
+						{
+							"bool": {
+								"filter": [],
+								"must": [],
+								"must_not": [],
+								"should": []
+							}
+						},
+						{
+							"bool": {
+								"filter": [],
+								"must": [],
+								"must_not": [],
+								"should": []
+							}
+						}
+					],
+					"must_not": [],
+					"should": []
+				}
+			},
+			"runtime_mappings": {},
+			"size": 0,
+			"timeout": "30000ms",
+			"track_total_hits": true
+		}`,
+		ExpectedResponse: `
+		{
+			"completion_time_in_millis": 1728635627258,
+			"expiration_time_in_millis": 1728635687254,
+			"id": "FlhaTzBhMkpQU3lLMmlzNHhBeU9FMHcbaUp3ZGNYdDNSaGF3STVFZ2xWY3RuQTo2MzU4",
+			"is_partial": false,
+			"is_running": false,
+			"response": {
+				"_shards": {
+					"failed": 0,
+					"skipped": 0,
+					"successful": 1,
+					"total": 1
+				},
+				"aggregations": {
+					"timeseries": {
+						"buckets": [
+							{
+								"a2": {
+									"value": 202.0
+								},
+								"doc_count": 202,
+								"key": 1728518400000,
+								"key_as_string": "2024-10-10T00:00:00.000Z"
+							}
+						],
+						"interval": "7d",
+						"meta": {
+							"dataViewId": "d3d7af60-4c81-11e8-b3d7-01146121b73d",
+							"indexPatternString": "kibana_sample_data_flights",
+							"intervalString": "54000000ms",
+							"normalized": true,
+							"panelId": "1a1d745d-0c21-4103-a2ae-df41d4fbd366",
+							"seriesId": "866fb08f-b9a4-43eb-a400-38ebb6c13aed",
+							"timeField": "timestamp"
+						}
+					}
+				},
+				"hits": {
+					"hits": [],
+					"max_score": null,
+					"total": {
+						"relation": "eq",
+						"value": 202
+					}
+				},
+				"timed_out": false,
+				"took": 4
+			},
+			"start_time_in_millis": 1728635627254
+		}`,
+		ExpectedPancakeResults: []model.QueryResultRow{
+			{Cols: []model.QueryResultCol{
+				model.NewQueryResultCol("aggr__timeseries__count", int64(202)),
+				model.NewQueryResultCol("metric__timeseries__61ca57f2-469d-11e7-af02-69e470af7417-denominator_col_0", int64(202)),
+				model.NewQueryResultCol("metric__timeseries__61ca57f2-469d-11e7-af02-69e470af7417-numerator_col_0", int64(0)),
+			}},
+		},
+		ExpectedPancakeSQL: `
+			SELECT count(*) AS "aggr__timeseries__count",
+			  countIf(True) AS
+			  "metric__timeseries__61ca57f2-469d-11e7-af02-69e470af7417-denominator_col_0",
+			  countIf(NOT ("xdr.result_code_str" = 'DIAMETER_SUCCESS')) AS
+			  "metric__timeseries__61ca57f2-469d-11e7-af02-69e470af7417-numerator_col_0"
```

Comment:
How this `61ca57f2-469d-11e7-af02-69e470af7417` is generated? Not sure if I recall this logic. I thought it should be just: `metric__timeseries__61ca57f2-469d-11e7-af02-69e470af7417-denominator_col_0`

Reply:
It's a normal aggregation name, results for this tests aren't correct yet.